x-pack/plugins/security_solution/public/onboarding/constants.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export const PAGE_CONTENT_WIDTH = '1150px';
+
+export enum OnboardingCardId {
+  integrations = 'integrations',
+  dashboards = 'dashboards',
+  rules = 'rules',
+  alerts = 'alerts',
+  assistant = 'asistant',
```

Comment:
typo: missed `s`, in `asistant`

Reply:
enums ftw 😉

Reply:
to be fair tho, string enums are also defined like this and the typo would still occur :)

Reply:
yes, what I mean is that I can change this value without having to worry about breaking anything.